cmd/utils/flags.go

```diff
@@ -659,7 +659,7 @@ var (
 	MetricsPortFlag = cli.IntFlag{
 		Name:  "metrics.port",
 		Usage: "Metrics HTTP server listening port",
-		Value: metrics.DefaultConfig.Port,
```

Comment:
then need remove `metrics.DefaultConfig.Port` field?

Reply:
https://github.com/erigontech/erigon/pull/12111 @AskAlexSharov here it is.